contracts/TablelandTables.sol

```diff
@@ -69,7 +69,8 @@ contract TablelandTables is
     );
 
     function runSQL(address caller, uint256 tableId, string memory statement) public {
-        require(caller == _msgSender(), "Tables: caller must be sender"); // temp, caller must be sender (later msg.sender could be a delegate)
+        // temp, caller must be sender or admin (later msg.sender could be a delegate)
+        require(caller != _msgSender() && !hasRole(DEFAULT_ADMIN_ROLE, _msgSender()), "Tables: caller must be sender or admin");
```

Comment:
Simple check to see if the caller is the sender, or if the sender has DEFAULT_ADMIN_ROLE.
This change allows the end to end tests to work, and development of the chess game to move forward.
Also related to https://www.notion.so/textile/Implement-on-behalf-signing-for-relayed-runSQL-in-the-validator-5db1097e259e4726846b95100c6ee71c

Reply:
👍 thanks for adding this!